llms/googleai/googleai_llm_test.go

```diff
@@ -0,0 +1,89 @@
+package googleai
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/joho/godotenv"
+	"github.com/stretchr/testify/require"
+	"github.com/tmc/langchaingo/llms"
+	"google.golang.org/api/option"
+)
+
+func TestMain(m *testing.M) {
+	// This loads the API key if it is a .env file
+	_ = godotenv.Load()
```

Comment:
We don't use `godotenv` elsewhere - let's just expect the env var to be set, like other tests do. The test should be skipped if the env var isn't set

Follow the example of `llms/openai/multicontent_test.go`